x-pack/filebeat/module/oracle/database_audit/ingest/pipeline.yml

```
@@ -37,21 +45,26 @@ processors:
   - script:
       source: "ctx.oracle.database_audit = ctx.oracle.database_audit.entrySet().stream().collect(Collectors.toMap(entry -> entry.getKey().toLowerCase(), Map.Entry::getValue));"
       lang: painless
+      if: ctx?.oracle?.database_audit != null
```

```suggestion
      if: ctx.oracle?.database_audit != null
```
(similar throughout)